components/database/sqlite/files/mapfile

```diff
@@ -27,6 +27,25 @@
 #
 $mapfile_version 2
 
+SYMBOL_VERSION sqlite_3.31.1 {
+    global:
+        sqlite3_column_origin_name;
```

Comment:
We miss a lot of public symbols in libsqlite3, introduced between 3.19 and 3.30.
I put what I've found here.